docs/dev/arch_decision.md

```diff
@@ -2,158 +2,6 @@
 
 The intention is to document deviations from a standard Model View Controller (MVC) design.
```

Comment:
@snaselj looks like it replaced this file, can you check why?

Reply:
Fixed by cleanup